src/components/viz/DataSeries/index.js

```diff
@@ -68,13 +70,41 @@ const GRAPH_MODES = {
 };
 
 
-const GRAPH_LABELS = {
-    [GRAPH.horizontalBarChart]: <span>{_ts('components.viz.dataSeries', 'horizontalBarChartLabel')}</span>,
-    [GRAPH.verticalBarChart]: <span>{_ts('components.viz.dataSeries', 'verticalBarChartLabel')}</span>,
-    [GRAPH.wordCloud]: <span>{_ts('components.viz.dataSeries', 'wordCloudLabel')}</span>,
-    [GRAPH.geo]: <span>{_ts('components.viz.dataSeries', 'geoLabel')}</span>,
+const GRAPH_DETAILS = {
+    [GRAPH.horizontalBarChart]: {
+        title: _ts('components.viz.dataSeries', 'horizontalBarChartLabel'),
+        iconName: iconNames.horizontalBar,
+    },
+    [GRAPH.verticalBarChart]: {
+        title: _ts('components.viz.dataSeries', 'verticalBarChartLabel'),
+        iconName: iconNames.verticalBar,
+    },
+    [GRAPH.wordCloud]: {
+        title: _ts('components.viz.dataSeries', 'wordCloudLabel'),
+        iconName: iconNames.word,
+    },
+    [GRAPH.geo]: {
+        title: _ts('components.viz.dataSeries', 'geoLabel'),
+        iconName: iconNames.globe,
+    },
 };
 
+const Tab = ({
+    icon,
+    onClick,
+    isActive,
+}) => (
+    <button
+        onClick={onClick}
```

Comment:
This is very risky, if you are using base <button>,
need to specify type of button `type="button"`
or it will submit the form